package.json

```diff
@@ -39,8 +34,21 @@
   },
   "homepage": "https://github.com/esm-bundle/rxjs#readme",
   "devDependencies": {
-    "esm-bundle-scripts": "1.2.0",
+    "@arkweid/lefthook": "^0.7.2",
```

Comment:
Why the switch from husky to lefthook?

Reply:
Just for fun. I wanted to try it out. It operates essentially the same, just faster.

Reply:
It also felt a little bit easier to migrate to vs Husky V5, if you'd prefer that I switch to husky v5 I will.

Reply:
I don't have a preference. Upgrading to husky 5 isn't hard if you use https://github.com/typicode/husky-4-to-5. But since you've already got it working with lefthook there's no need to change.